ansible/roles/atmo-ssh-setup/vars/CentOS.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+SSH_SERVICE: sshd
+
+PORT_ENTRY_REGEXP: "^# default value."
+
+PORT_ENTRY_LINE: "# default value.\nPort {{ SSH_PORT }}"
```

Comment:
My reasoning for this is that sometimes the "Port" line is commented out, so I had lineinfile to remove the port line if it is commented or uncommented and then add it again later. The reason for matching the weird comments is so that the Port entry is in the same place it would regularly be in.

However, after looking into it some more, it doesnt look like the Port line is often commented out by default so I can probably drop the weirdness. If the "Port" line is not found, is it fine to just append the Port entry to the end of file?